projects/tn-ro-manual.diff

```diff
@@ -1068,7 +1077,7 @@ index 3042dc1..632ed76 100644
              </property>
          </source>
          <target>
-@@ -5361,21 +5430,6 @@ Für alle anderen `bauwerksfunktion`en, Referenzen auf andere Objekte oder auch
+@@ -5621,21 +5690,6 @@ Für alle anderen `bauwerksfunktion`en, Referenzen auf andere Objekte oder auch
```

Comment:
Looks like there is an accent too much

Reply:
Do you mean the two backticks? They are used in Markdown to format a textblock as code and are often used to highlight variable names. In this case
```
`bauwerksfunktion`en
```
displays as `bauwerksfunktion`en.

Reply:
I see - yes that is what I meant. I must have missed the first one. Thanks for clarification.